src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/DisplayDimensions/DisplayDimensions.xaml.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.DisplayDimensions
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        name: "Display dimensions",
+        category: "Layers",
+        description: "Display dimension features from a mobile map package.",
+        instructions: "When the sample loads, it will automatically display the map containing dimension features from the mobile map package. The name of the dimension layer containing the dimension features is displayed in the controls box. Control the visibility of the dimension layer with the \"Dimension Layer visibility\" check box, and apply a definition expression to show dimensions of greater than or equal to 450m in length using the \"Definition Expression\" checkbox.",
+        tags: new[] { "dimension", "layer", "mmpk", "mobile map package", "utility", "Featured" })]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData("f5ff6f5556a945bca87ca513b8729a1e")]
+    public partial class DisplayDimensions
+    {
+        // Dimension layer, the operational layer. 
```

Comment:
This might be misleading since there could be multiple operational layers. I think the pattern that's been used in other samples is `Hold a reference to the Dimension layer for use in event handlers` or something to that effect.